src/main/java/기초별찍기C/ProgramC.java

```diff
@@ -0,0 +1,71 @@
+package 기초별찍기C;
+
+import java.util.Scanner;
+
+/**
+ * 2. 기초 별찍기 c. 숫자를 입력받고, 해당 숫자의 크기의 마름모를 출력하는 프로그램을 만들어주세요.
+ *    (숫자 크기를 마름모의 전체 라인 개수로 설정했음)
+ */
+public class ProgramC {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String input;
+        Verification vrfc = new Verification();
+
+        do {
+            System.out.println("3이상의 정수를 입력해주세요.");
+            System.out.println("입력값이 마름모의 전체 라인 개수 입니다. 3이상의 짝수인 경우 1을 더해 홀수로 바꿔 마름모를 만듭니다.");
+            System.out.print(">> ");
+            input = scan.next();
+        } while (!vrfc.isValid(input));
+
+        int inputNumber = Integer.parseInt(input);
+
+        //입력값이 짝수면 1을 더한 홀수로 바꿔서 그 값을 마름모의 전체 라인 개수로 계산
+        int n = (inputNumber%2 == 0) ? inputNumber+1 : inputNumber;
+
+        //마름모의 윗 부분. 마름모의 라인 1 ~ (n+1)/2 까지의 삼각형
+        for (int i = 0; i < (n+1)/2; i++){
+            for (int j = 0; j < ((n+1)/2)-1 + (i*(-1)); j++) {
+                System.out.print(" ");
+            }
+            for (int j = 0; j < 1 + (i*2); j++){
+                System.out.print("*");
+            }
+            System.out.println();
+        }
```

Comment:
```suggestion
        int halfRowIndex = (n+1)/2;
        for (int i = 0; i < halfRowIndex; i++){
            int whiteSpaceCount = ((n+1)/2)-1 + (i*(-1));
            int starCount = 1 + (i*2);
            for (int j = 0; j < whiteSpaceCount; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < starCount; j++){
                System.out.print("*");
            }
            System.out.println();
        }
```

```
n 
(n+1)/2
((n+1)/2)-1 + (i*(-1))
```
반복문에서 사용되는 위와 같은 수식들이 어떤값을 의미하는지 변수를 이용해 표현한다면 더 가독성이 좋아질거 같네요~
i, j, n 등도 변수의 의미를 생각하며 네이밍 해봅시다.